openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -0,0 +1,305 @@
+                                                <td class="value">{{ criterion.total_value }}</td>
+                                            </tr>
+                                        {% endif %}
+                                    {% endfor %}
+                                {% endfor %}
+                                </tbody>
+                            </table>
+                            <h4 class="title--sub">{% trans "Overall Feedback" %}</h4>
+                            <div class="student__answer__display__content">
+                                {{ assessment.feedback|linebreaks }}
+                            </div>
+                        {% endwith %}
+                    {%  endfor %}
+                </div>
+            </div>
+        </div>
+    {% endif %}
+
+    {% if self_assessment %}
+        <div class="staff-info__status ui-staff__content__section wrapper--ui--collapse">
+            <div class="ui-staff ui-toggle-visibility is--collapsed">
+                <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+                    <i class="icon fa fa-caret-right"></i>
+                    <span>{% trans "Learner's Self Assessment" %}</span>
+                </h2>
+                <div class="ui-toggle-visibility__content">
+                    <table class="staff-info__status__table" summary="{% trans "Self Assessment" %}">
+                        <thead>
+                        <tr>
+                            <th abbr="Criterion" scope="col">{% trans "Criterion" %}</th>
+                            <th abbr="Selected Option" scope="col">{% trans "Selected Option" %}</th>
+                            <th abbr="Points" scope="col">{% trans "Points" %}</th>
+                            <th abbr="Points Possible" scope="col">{% trans "Points Possible" %}</th>
+                        </tr>
+                        </thead>
+
+                        <tbody>
+                        {% for criterion in rubric_criteria %}
+                            {% for part in self_assessment.parts %}
+                                {% if part.option.criterion.name == criterion.name %}
+                                    <tr>
+                                        <td class="label">{{ criterion.label }}</td>
+                                        <td class="value">{{ part.option.label }}</td>
+                                        <td class="value">{{ part.option.points }}</td>
+                                        <td class="value">{{ criterion.total_value }}</td>
+                                    </tr>
+                                {% endif %}
+                            {% endfor %}
+                        {% endfor %}
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+    {% endif %}
+
+    {% if example_based_assessment %}
```

Comment:
@explorerleslie or @catong  Do you know how to configure example based assessments? We'd like to make sure that the HTML didn't break, but @andy-armstrong (and I) don't really understand example based assessments.